step_definitions/geography/geography_landing_page.js

```diff
@@ -0,0 +1,53 @@
+var {client} = require('nightwatch-cucumber');
+var {defineSupportCode} = require('cucumber');
+var http = require('http');
+
+var geographyLandingPage = client.page.geographyLandingPage();
+
+defineSupportCode(({Given, Then, When}) => {
+    /*
+    Reused across scenarios
+    */
+    Given(/^I go to a geography landing page/, () => {
+        return geographyLandingPage
+        
+            .navigate()
+            .waitForLoad();
+    });
+
+
+    Then(/^I can see the 'Area Types' dimension/, () => {
+        return geographyLandingPage
+            .expect.element('@areatypes').to.be.visible;
+    });
+
+    // Then(/^I can see the description for the 'Area Types'/, () => {
+        
+    //     client.elements('css selector', '.area-type', function(result){
+    //         console.log("test 12345678", result.value);
+    //         result.value.forEach(element => {
+    //             console.log("test", element);
+                
+                
+    //         });
+            
+    //     });
+    //     return geographyLandingPage
+        
+    // });
+    Then(/^I can see the description for the 'Area Type id 0'/, () => {
```

Comment:
Similar to above, checking three specific area types is quite fragile, what if the name changes slightly or we add another one? Perhaps just checking that a valid title renders for each area type is more resilient.